tests/Functional/EnvCommandsTest.php

```diff
@@ -74,13 +91,56 @@ public function testCloneContentCommand()
      *
      * @group env
      * @group long
+     * @group wordpress
      */
     public function testCloneContentForWordpressCommand()
     {
         // 1. Ensure that the multidev environment exists
-        $this->ensureSiteEnvironment($this->getSiteName("wordpress"), $this->getMdEnv());
-        $this->terminus(sprintf('env:clone-content %s.%s %s', $this->getSiteName(), 'dev', $this->getMdEnv()));
-        $this->deleteSiteEnvironment($this->getSiteName("wordpress"), $this->getMdEnv());
+        $this->ensureSiteEnvironment(
+            $this->getSiteName("wordpress"),
+            $this->getMdEnv()
+        );
+        $this->terminus(
+            sprintf(
+                'env:clone-content %s.%s %s',
```

Comment:
Nit: It might be helpful to do 
```suggestion
                'env:clone-content %1$s.%2$s %3$s',
```
just to be more explicit about what goes where. Functionally, I realize it doesn't change anything, but it looks weird to me to see three `%s`s in a row.

🔢 This comment applies elsewhere in the PR where the same pattern is used.

Reply:
I didn't realize you could do this in PHP

Reply:
Sure can! :D